pkg/eventhandler/event_handler.go

```diff
@@ -66,9 +66,10 @@ func NewFromConfig(ctx context.Context, cli client.Client, actionMeta v1alpha1.A
 		// TODO: use handler to handle
 		// Apply filters
 		context := map[string]interface{}{
-			"event":     event,
-			"data":      data,
-			"timestamp": time.Now().Format(time.RFC3339),
+			"sourceType": sourceType,
+			"event":      event,
+			"data":       data,
+			"timestamp":  metav1.Now(),
```

Comment:
Why delete the time format?

Reply:
https://github.com/kubevela/kube-trigger/blob/6582e7595db61578d08bc3ac03ffef77c6a4319c/api/v1alpha1/eventlistener_types.go#L51

I made it to use `metav1` to keep it consistent with the definition in `api`. The format should still be there, which is the same as the one used in k8s resources like this `2023-03-08T13:33:05Z`

Reply:
This timestamp is in the context which users can get in CUE, I think it's better to make it readable.

Reply:
yes it makes sense